tools/meson.build

```diff
@@ -0,0 +1,19 @@
+foreach e : [
+    ['pvxvct', ['./pvxvct.cpp']],
+    ['pvxinfo', ['./info.cpp']],
+    ['pvxmonitor', ['./monitor.cpp']],
+    ['pvxput', ['./put.cpp']],
+    ['pvxcall', ['./call.cpp']],
+    ['pvxlist', ['./list.cpp']],
+    ['pvxmshim', ['./mshim.cpp']],
+  ]
+  executable(
+    e[0],
+    e[1],
```

Comment:
You're using `foreach e` here, in tools, and `foreach t` in examples. Aren't they switched around?

Reply:
I'm using `e` for executable here, and `t` for tests elsewhere. I should probably change to `e` in `examples`, since they're not really "tests"

Reply:
Sounds good to me!